content/yaml/yaml.md

```diff
@@ -0,0 +1,225 @@
+        - push
+        - pull_request
+        - tag
+      branch_patterns:       # Include or exclude watched branches
+        - pattern: '*'
+          include: true
+          source: true
+        - pattern: excluded-target
+          include: false
+          source: false
+        - pattern: included-source
+          include: true
+          source: true
+
+
+### Scripts
+
+Scripts specify what kind of application is built.  This is where you can specify the commands to test, build and code sign your project. There are example scripts available for building a [Flutter application](./building-a-flutter-app/), [React Native application](./building-a-react-native-app/), [native Android application](./building-a-native-android-app/) or a [native iOS application](./building-a-native-ios-app/).
+
+### Testing
+
+There are two types of tests that users can run when developing mobile apps, unit tests (for testing code) and instrumentation tests (for testing the UI and the application in general). These tests take place in a simulator (iOS) or emulator (Android), depending on the platform. Examples of testing are available [here](./testing/).
+
+### Artifacts
+
+Configure the paths and names of the artifacts you would like to use in the following steps, e.g. for publishing, or have available for download on the build page. All paths are relative to the clone directory, but absolute paths are supported as well. You can also use environment variables in artifact patterns.
+
+    artifacts:
+      - build/**/outputs/**/*.apk                   # relative path for a project in root directory
+      - subfolder_name/build/**/outputs/**/*.apk    # relative path for a project in subfolder
+      - build/**/outputs/**/*.aab
+      - build/**/outputs/**/mapping.txt
+      - build/ios/ipa/*.ipa
+      - /tmp/xcodebuild_logs/*.log
+      - flutter_drive.log
+
+There are several things to keep in mind about patterns:
+* The pattern can match several files or folders. If it picks up files or folders with the same name, the top level file or folder name will be suffixed with `_{number}`.
+* If one of the patterns includes another pattern, duplicate artifacts are not created.
+* `apk`, `aab`, `ipa`, `aar`, `app`, proguard mapping (`mapping.txt`), `flutter_drive.log`, `jar`, `zip`, `xarchive` and `dSYM.zip` files will be available as separate items in the Artifacts section on the build page. The rest of the artifacts will be included in an archive with the following name pattern: `{project-name}_{version}_artifacts.zip`.
+
+### Publishing
+
+All the apps that run on iOS devices need to be signed by trusted developers with a valid certificate from Apple. This confirms the author of the code and guarantees that the code has not been tampered with since it was signed. After code signing, the generated artifacts can be easily published to external services. Both code singing and publishing is explained [here](./distribution/) in more detail.
```

Comment:
Code signing is not relevant just for iOS apps but also for Android apps that are built for release. Also, the emphasis in this section should be more about distribution options since this is what is configured here. Code signing settings are defined elsewhere in yaml.